gpkit/solution_array.py

```diff
@@ -276,104 +269,3 @@ def results_table(data, title, minval=0, printunits=True, fixedcols=True,
                  ["\\toprule"] + [title + " & Value & Units \\\\"] + ["\\midrule"] +
                  [''.join(l) for l in lines] + ["\\bottomrule"] + ["\\end{longtable}}"] + [""])
     return lines
-
-
-def parse_result(result, constants, beforesubs, sweep={}, linkedsweep={},
-                 freevar_sensitivity_tolerance=1e-4,
-                 localmodel_sensitivity_requirement=0.1):
-    "Parses a GP-like result dict into a SolutionArray-like dict."
-    cost = result["cost"]
-    freevariables = dict(result["variables"])
-    sweepvariables = {var: val for var, val in constants.items()
-                      if var in sweep or var in linkedsweep}
-    constants = {var: val for var, val in constants.items()
-                 if var not in sweepvariables}
-    variables = dict(freevariables)
-    variables.update(constants)
-    variables.update(sweepvariables)
-    sensitivities = dict(result["sensitivities"])
-
-    # Remap monomials after substitution and simplification.
-    #  The monomial sensitivities from the GP/SP are in terms of this
-    #  smaller post-substitution list of monomials, so we need to map that
-    #  back to the pre-substitution list.
-    #
-    #  Each "smap" is a list of HashVectors (mmaps),
-    #    whose keys are monomial indexes pre-substitution,
-    #    and whose values are the percentage of the simplified monomial's
-    #    coefficient that came from that particular parent
```

Comment:
did the nice inline comment above get turned into a docstring somewhere, if any of it is still relevant?